src/drivers/navio_rgbled/test/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ add_executable(navio_rgbled_test
 	test.cpp
 	)
 
-target_link_libraries(navio_rgbled_test
+target_link_libraries(navio_rgbled_test PRIVATE
```

Comment:
Why private?

Reply:
These are dependencies of navio_rgbled_test. With target_link_libraries you can't mix usage of PRIVATE/PUBLIC/INTERFACE with nothing.